loki/README.md

```diff
@@ -172,12 +135,88 @@ kubectl -n ${KYMA_NS} get pod -l app=loki,release=${HELM_LOKI_RELEASE}
      'time={NANOSECONDS}'
    ```
 
+## Install Grafana
+
+Because Grafana provides a very good Loki integration, you might want to install it as well.
+
+### Installation
+
+1. To deploy Grafana, run:
+
+   ```bash
+   helm upgrade --install --create-namespace -n ${KYMA_NS} grafana grafana/grafana -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/grafana-values.yaml
+   ```
+
+1. To enable Loki as Grafana Datasource, run:
+   ```bash
+   cat <<EOF | kubectl apply -n ${KYMA_NS} -f -
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: grafana-loki-datasource
+     labels:
+       grafana_datasource: "1"
+   data:
+      loki-datasource.yaml: |-
+         apiVersion: 1
+         datasources:
+         - name: Loki
+           type: loki
+           access: proxy
+           url: "http://${HELM_LOKI_RELEASE}:3100"
+           version: 1
+           isDefault: false
+           jsonData: {}
+   EOF
+   ```
+
+1. To access the Grafana UI with kubectl port forwarding, run:
+
+   ```bash
+   kubectl get secret --namespace mlp grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+   ```
+   
+   to get the password for the access, then run:
+   ```bash
+   kubectl -n ${KYMA_NS} port-forward svc/grafana 3000:80
+   ```
+   
+   Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
+  
+## Exposure
```

Comment:
```suggestion
## Grafana Exposure

### Expose Grafana
```

All previous sections had a h2 headline first and then the instructions below the h3 headline. Why is it different here?